demo/onBoarding/demo.js

```diff
@@ -0,0 +1,155 @@
+angular.module('demo', ['znk.infra-web-app.onBoarding', 'ngSanitize'])
+    .config(function ($translateProvider, znkAnalyticsSrvProvider, $urlRouterProvider, InfraConfigSrvProvider, OnBoardingServiceProvider, DiagnosticIntroSrvProvider, UserGoalsServiceProvider) {
+
+        $urlRouterProvider.otherwise('/onBoarding');
+
+        $translateProvider.useLoader('$translatePartialLoader', {
+                urlTemplate: '/{part}/locale/{lang}.json'
+            })
+            .preferredLanguage('en')
+            .useSanitizeValueStrategy('sanitize');
+
+        znkAnalyticsSrvProvider.setEventsHandler(function () {
+            return {
+                eventTrack: angular.noop,
+                timeTrack: angular.noop,
+                pageTrack: angular.noop,
+                setUsername: angular.noop,
+                setUserProperties: angular.noop
+            };
+        });
+
+        var storageFake = ['$q', function($q) {
+            return {
+                get: function(path, data) {
+                    return $q.when(data || {});
+                },
+                set: function(path, data) {
+                    return $q.when(data || {});
+                },
+                variables: {
+                    appUserSpacePath: ''
+                }
+            }
+        }];
+
+        InfraConfigSrvProvider.setStorages(storageFake, storageFake);
+
+        InfraConfigSrvProvider.setUserDataFn(['AuthService', function (AuthService) {
+            return AuthService.getAuth();
+        }]);
+
+        OnBoardingServiceProvider.settings = {
+           showSchoolStep: true
+        };
+
+        DiagnosticIntroSrvProvider.setConfigMapFn(['ENV', function(ENV) {
```

Comment:
Can we reduce the size of the config ? like providing only subject name and rip it for the other fields ?